src/hooks/useManagedChangeableValue.ts

```
@@ -2,19 +2,23 @@ import {
   ChangeEvent,
   ChangeEventHandler,
   InputHTMLAttributes,
+  SelectHTMLAttributes,
+  TextareaHTMLAttributes,
   useReducer,
 } from 'react';
 
-type ValueType = InputHTMLAttributes<HTMLInputElement>['value'];
+type HTMLElementWithValue = HTMLInputElement | HTMLTextAreaElement | HTMLSelectElement;
```

ESLint

[error] Replace `·HTMLInputElement·|·HTMLTextAreaElement` with `⏎··|·HTMLInputElement⏎··|·HTMLTextAreaElement⏎·`

(prettier/prettier)